File: src/OK_GER05.java
import java.util.*;
public class OK_GER05 {
   public static void main(String []args) {
      new Program().run();
   }
}
class Scan {
   public static Scanner imputValue = new Scanner(System.in);
}
class A {
   public void m() {
      int a, b, c, d, e, f;
      a = Scan.imputValue.nextInt();
      b = Scan.imputValue.nextInt();
      c = Scan.imputValue.nextInt();
      d = Scan.imputValue.nextInt();
      e = Scan.imputValue.nextInt();
      f = Scan.imputValue.nextInt();
      System.out.print("" + a);
      System.out.print("" + b);
      System.out.print("" + c);
      System.out.print("" + d);
      System.out.print("" + e);
      System.out.print("" + f);
   }
}
class Program {
   public void run() {
      A a;
      System.out.println("" + "");
      System.out.println("" + "Ok-ger05");
      System.out.println("" + "The output should be what you give as input.");
      System.out.println("" + "Type in six numbers");
      a = new A();
      a.m();
   }
}

File: src/OK_GER05.c
#include<malloc.h>
#include<stdlib.h>
#include<stdio.h>
#include<string.h>

typedef int boolean;

#define true 1
#define false 0

int readInt() {
   int _n;
   char __s[512];
   gets(__s);
   sscanf(__s, "%d", &_n);
   return _n;
}

char *readString() {
   char s[512];
   gets(s);
   char *ret = malloc(strlen(s) + 1);
   strcpy(ret, s);
   return ret;
}

char *intToString(int _n) {
   char *ret = malloc(512);
   sprintf(ret, "%d", _n);
   return ret;
}

char *concatStrings(char string1[], char string2[]) {
   char *ret = malloc(strlen(string1) + strlen(string2) + 1);
   strcpy(ret,string1);
   strcat(ret,string2);
   return ret;
}

typedef void (*Func)();

typedef struct _St_A {
   Func *vt;
}_class_A;

_class_A *new_A(void);

void _A_m(_class_A *self) {
   int _a, _b, _c, _d, _e, _f;
   _a = readInt();
   _b = readInt();
   _c = readInt();
   _d = readInt();
   _e = readInt();
   _f = readInt();
   printf("%d", _a);
   printf("%d", _b);
   printf("%d", _c);
   printf("%d", _d);
   printf("%d", _e);
   printf("%d", _f);
}

Func VTclass_A[] = {
   (void(*)())_A_m
};

_class_A *new_A() {
   _class_A *t;
   
   if((t = malloc(sizeof(_class_A))) != NULL) {
      t->vt = VTclass_A;
   }
   
   return t;
}

typedef struct _St_Program {
   Func *vt;
}_class_Program;

_class_Program *new_Program(void);

void _Program_run(_class_Program *self) {
   _class_A *_a;
   printf("%s\n", "");
   printf("%s\n", "Ok-ger05");
   printf("%s\n", "The output should be what you give as input.");
   printf("%s\n", "Type in six numbers");
   _a = new_A();
   ((void (*)(_class_A* ))_a->vt[(sizeof(VTclass_A)/sizeof(VTclass_A[0])) - 1 - 0])(_a);
}

Func VTclass_Program[] = {
   (void(*)())_Program_run
};

_class_Program *new_Program() {
   _class_Program *t;
   
   if((t = malloc(sizeof(_class_Program))) != NULL) {
      t->vt = VTclass_Program;
   }
   
   return t;
}

int main() {
   _class_Program *program;
   program = new_Program();
   ((void (*)(_class_Program*)) program->vt[0])(program);
   return 0;
}
